parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageReadStore.java

```diff
@@ -168,12 +168,7 @@ public DataPage visit(DataPageV1 dataPageV1) {
                   decompressedBuffer,
                   dataPageV1.getUncompressedSize());
               setDecompressMetrics(bytes, start);
-
-              // HACKY: sometimes we need to do `flip` because the position of output bytebuffer is
```

Comment:
Since we already defined the contract which actually requires a flip to be invoked here, I don't think we need the comment, especially not calling this hacky.